tests/WebAppCallsWebApiCallsGraph/Client/TodoListClient.csproj

```diff
@@ -23,7 +23,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.DataProtection.Abstractions" Version="3.1.1" />
-    <PackageReference Include="Microsoft.Graph" Version="1.16.0" />
```

Comment:
Microsoft.Graph is now included in Microsoft.Identity.Web. We don't need to reference it any longer in the samples